src/NHibernate.Test/Linq/WhereSubqueryTests.cs

```diff
@@ -579,6 +618,24 @@ public void OrdersWithSubquery11()
 			Assert.That(productsNotInLargestOrders.Count, Is.EqualTo(49), nameof(productsNotInLargestOrders));
 		}
 
+		[Test]
+		public void OrdersWithSubquery11A()
+		{
+			//if (Dialect is MsSqlCeDialect)
+			//	Assert.Ignore("MS SQL CE does not support sorting on a subquery.");
+
```

Comment:
```suggestion
```

Reply:
And async regen is missing

Reply:
I'll remove the commented code after the MsSqlCeDialect finished.

Reply:
Addressed.